dashboard/config/blocks/GamelabJr/gamelab_patrollingUpDown.json

```diff
@@ -0,0 +1,13 @@
+{
+  "category": "SL1",
```

Comment:
What does this category mean?

Reply:
No idea but it's the [gamelab_draggable](https://github.com/code-dot-org/code-dot-org/blob/219761a42ed9315f0eba5fb7f32475c2ce679f48/dashboard/config/blocks/GamelabJr/gamelab_draggable.json#L2) is in (I cloned both of these blocks from that one). Happy to change it if there's a better category.